src/passes/stringify-walker.h

```diff
@@ -0,0 +1,36 @@
+#ifndef passes_stringify_walker_module_h
+#define passes_stringify_walker_module_h
+
+#include <queue>
+
+namespace wasm {
+
+struct StringifyWalker
```

Comment:
It will be useful to make this a CRTP utility, i.e. to make this a template that takes a `SubType` parameter like other generic walkers. This will let us define multiple concrete subclasses with different implementations of `visitExpression`, `visitControlFlow`, etc. for use in different parts of outlining and in testing.